samples/sensor/mcp9600/README.rst

```diff
@@ -0,0 +1,49 @@
+.. _mcp9600:
+
+mcp9600 Temperature Sensor
```

Comment:
Please split the driver and sample application into two commits

Reply:
OK easier for me to close this one and create two new ones.